include/seastar/net/socket_defs.hh

```diff
@@ -123,6 +133,14 @@ struct ipv6_addr {
     }
 };
 
+struct ud_addr {
+    std::filesystem::path sfile_;
+
+    explicit ud_addr(const std::string&);
```

Comment:
the implementation of this constructor is not defined.

Reply:
fixed.